src/Folder.jsx

```diff
@@ -1,28 +1,104 @@
+import { useState, useEffect } from 'react';
+
+import { getFolderList, getLink, getLinkDetail } from './apis/api';
+import shareImg from './assets/folder/share.svg';
+import renameImg from './assets/folder/pen.svg';
+import deleteImg from './assets/folder/delete.svg';
+
+import FolderAddBtn from './components/FolderAddBtn';
 import SearchBar from './components/SearchBar';
 import Card from './components/Card';
 import './Folder.css';
 
-const Folder = ({items}) => {
-  if(!items || !items.links) {
-    return (
-      <div>파트 1에서 만들었던 랜딩 페이지 적용</div>
-    )
+const FOLDER_EDITS = Object.freeze([
+  { src: shareImg, name: "공유" },
+  { src: renameImg, name: "이름변경" },
+  { src: deleteImg, name: "삭제" },
+]);
+
+const Folder = () => {
+  const [folderLists, setFolderLists] = useState([]); // 이건 부모 컴포넌트에서 넘겨주는 게 좋을 듯
+  const [folders, setFolders] = useState([]);
+  const [curfolderList, setCurFolderList] = useState();
```

Comment:
사용하는쪽을 보니까 이 상태값은 문자열 데이터가 저장될 것으로 보이는데요.
그러면 빈 문자열로 초기화를 해주시는게 좋습니다.
```suggestion
  const [curfolderList, setCurFolderList] = useState('');
```

Reply:
그리고 여기서는 '전체'로 초기화를 해주는게 일반적인 동작일 것 같아요.